network/src/commonMain/kotlin/com/wire/kalium/network/api/conversation/CreateConversationRequest.kt

```diff
@@ -15,7 +15,7 @@ data class CreateConversationRequest(
     // the name is optional in swagger but this should not be the case
     // since there is endpoint for one2one and self conversations
```

Comment:
```suggestion
    // the name is optional in swagger but this should not be the case
    // since there is endpoint for one2one and self conversations
```

Let's delete this comment now that name is optional.